lib/ipmi_backend_utils.pm

```diff
@@ -146,7 +147,19 @@ sub umount_installation_disk {
 }
 
 sub get_installation_partition {
-    my $partition = script_output("fdisk -l | grep \"^\/dev\/sda\.\*\\\*\" | cut -d ' ' -f 1");
+    my $partition = '';
+    if (sle_version_at_least('15')) {
+        my $y2log_file                = '/var/log/YaST2/y2log';
+        my $root_partition_commit_msg = script_output("grep \"Commit Action \\\"Adding mount point / of .* to /etc/fstab\" $y2log_file");
```

Comment:
Reading out that information from a logfile might be a workaround for the time being but I doubt this can be more robust than the fdisk approach.

Reply:
fdisk way does not work now.

Reply:
Is this a bug? Then you should use record_soft_fail

Reply:
I am not sure, but I consulted other guys , some say sle15 changes the behavior.
Please note that when running this code to detect the partition with the final installed os, it is when the actual installation finishes but before the system really boot to it. 
Although we did not use y2log way before, but I believe this should be robust, because y2log is the installation log, and we are sure that the mount information should be written to /etc/fstab. So with this commit msg, it should work well.